sdk/core/azure-core-test/src/test_proxy_policy.cpp

```diff
@@ -42,14 +42,21 @@ std::unique_ptr<RawResponse> TestProxyPolicy::Send(
   {
     // This is a download with keep connection open. Let's switch the request
     redirectRequest = Azure::Core::Http::Request(
-        request.GetMethod(), Azure::Core::Url(m_testProxy->GetTestProxy()), false);
+        request.GetMethod(),
+        Azure::Core::Url(m_testProxy->GetTestProxy()),
+        request.GetBodyStream(),
+        false);
   }
 
   redirectRequest.GetUrl().SetPath(request.GetUrl().GetPath());
 
   // Copy all headers
```

Comment:
Update comment to explain why you're redacting "host"?

Reply:
@LarryOsterman I'm not redacting `Host`. It's to fix a bug. I added explanation in the description of the PR.